spec/forms/hyrax/forms/permission_template_form_spec.rb

```diff
@@ -195,12 +195,14 @@
 
     context "when a new workflow has been chosen" do
       it "gives the managers workflow roles" do
+        expect(form).not_to receive(:update_permission_template)
```

Comment:
I agree. I thought it was a rather worthless test to add, which was why I didn't add it initially. But since you requested it, it's there now and I'm not particularly inclined to remove it again. I think this one-line fix has taken more of my time than it was worth and there are more important things to work on.